recipes/artery-font-format/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
```

Comment:
Please see https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases

Reply:
Thank you for the link  👍 
Since the library seems to have a version but no tag, I opened an issue. I'll use this procedure if I get no answer or a negative one.